lib/vcap_env.rb

```diff
@@ -0,0 +1,13 @@
+require 'json'
+
+# Parses ENV['VCAP_SERVICES'] or similar JSON and sets ENV variables for user-provided:credentials
+module VcapEnv
+  def self.set_env(vcap_json)
+    vcap = JSON.parse(vcap_json)
+    env = vcap["user-provided"].find { |service| service["name"] == "compliance-viewer-env"  }
```

Comment:
Yep, Roger.  I'll probably just merge this myself a little later today.   The merge will trigger the first attempt at CD in the production env.